### ISSUE TITLE
[Tooltip] Icon buttons with tooltips use the wrong aria label

### ISSUE THREAD

**Issue by jeffgammon** [angular/material2#4322]
#### Bug, feature request, or proposal:
Bug

#### What is the expected behavior?
We expect that aria-label for the button should be set from the md-tooltip, when present, rather than using the ID of the icon, which may or may not accurately represent the button's function.

#### What is the current behavior?
aria-label on the md-icon is the only aria-label set, which is what screen readers pick up and announce for that button.

#### What are the steps to reproduce?
1. Create a simple md-icon-button with an md-tooltip as shown here:
https://plnkr.co/edit/XyQdNCbGrBgWhPXEy6Kd?p=preview

Start NVDA screen reader and tab through that button. It reads "add box button".

#### What is the use-case or motivation for changing an existing behavior?
Blind users would like to know the purpose of the button, which is shown in the tooltip for non-blind users. The icon ID is frequently not the button's purpose, or may be only partially correct. In my plunkr, the words "add box" are announced by screen reader, which is confusing to blind users. We would like the blind user to hear "add some stuff" in this plunkr example.

#### Which versions of Angular, Material, OS, browsers are affected?
Material: 2.0.0-beta.3, 
I've tested NVDA 2017.1 on Firefox 53.0  and Chrome 57

**Comment by jelbourn** [comment]
@andrewseguin The changes I plan to make to md-icon will affect this issue

**Comment by andrewseguin** [comment]
@jelbourn Can you verify that this is fixed?

**Comment by jelbourn** [comment]
Icons no longer set `aria-label` to anything. Tooltips also intentionally do not set `aria-label` on the element they decorate, instead augmenting an existing label with `aria-describedby`. A button with no text content and a tooltip still needs an `aria-label`.
